bin/terminus.php

```diff
@@ -34,15 +39,26 @@
 $output = new ConsoleOutput();
 $container = Robo::createDefaultContainer($input, $output, $application, $config);
 
+$container->share('request', Request::class);
+$container->inflector(RequestAwareInterface::class)
+    ->invokeMethod('setRequest', ['request']);
+
 $container->share('fileCache', FileCache::class);
+
 $container->share('session', Session::class)
     ->withArgument('fileCache');
 $container->inflector(SessionAwareInterface::class)
     ->invokeMethod('setSession', ['session']);
+
 $container->share('sites', Sites::class);
 $container->inflector(SiteAwareInterface::class)
     ->invokeMethod('setSites', ['sites']);
 
+// Add our models
+$container->add(User::class);
```

Comment:
We should be consistent -- this is not to say that we would have to stop using aliases everywhere in order to use class or interface names some places -- but we should come up with an unambiguous rule / guideline about when to forgo the alias, and when to use it.